app.py

```diff
@@ -17,92 +46,222 @@ def create_folder(tpost):
     
     return fpath
 
+def append_to_legacy_rename_script_file(folder, file_path_legacy, file_path):
+    # todo: this should detect the host os and write the appropriate script type instead of windows batch
+    if config.write_legacy_rename_script and file_path_legacy != file_path:

```

Comment:
Support for writing a "legacy rename script" that can be optionally run to update the older short filenames to the new longer ones. No renaming or other file management is done automatically.